### DISCUSSION TITLE
Segment to display PowerShell Version

### DISCUSSION THREAD

**Discussion by ViciousMagician** [JanDeDobbeleer/oh-my-posh#894] [Troubleshoot] [answered]
Is there a way to display the version of powershell?

**Comment by ViciousMagician** [comment, marked as answer]
Nevermind, i asked too early.  My solution below in case anyone else is looking for an answer


Add this to your $PROFILE
`$env:POSH=$PSVersionTable.PSVersion.ToString()`


Add this segment to your theme
`		{
		  "type": "envvar",
		  "style": "powerline",
		  "powerline_symbol": "\uE0B0",
		  "foreground": "#ffffff",
		  "background": "#0077c2",
		  "properties": {
			"var_name": "POSH"
		  }
`

**Reply by sakabug** [reply]
[Here POSH holds the custom path string](https://i.imgur.com/Iw9sYah.png)
